specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/2020-01-15-generalAvailability/CommunicationIdentity.json

```diff
@@ -0,0 +1,264 @@
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Requesting scopes for the new token.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CommunicationTokenRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CommunicationIdentityToken"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Issue Token": {
+            "$ref": "./examples/IssueToken.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CommunicationIdentity": {
+      "description": "A communication identity.",
+      "required": [
+        "id"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Identifier of the identity.",
+          "type": "string"
+        }
+      }
+    },
+    "CommunicationIdentityUpdateRequest": {
+      "type": "object",
+      "properties": {
+        "tokensValidFrom": {
+          "format": "date-time",
+          "description": "All tokens that are issued prior to this time will be revoked.",
+          "type": "string"
+        }
+      }
+    },
+    "CommunicationTokenRequest": {
+      "required": [
+        "scopes"
+      ],
+      "type": "object",
+      "properties": {
+        "scopes": {
+          "description": "List of scopes attached to the token.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "CommunicationIdentityToken": {
+      "required": [
+        "expiresOn",
+        "id",
+        "token"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Identifier of the identity owning the token.",
+          "type": "string"
+        },
+        "token": {
+          "description": "The token issued for the identity.",
+          "type": "string"
+        },
+        "expiresOn": {
+          "format": "date-time",
+          "description": "The expiry time of the token.",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "name": "endpoint",
+        "description": "Auth and Identity endpoint",
```

Comment:
"The communication resource, for example https://my-resource.communication.azure.com"

Reply:
Updated